src/languages/typescript/typescript.ts

```diff
@@ -81,6 +81,9 @@ export const TypescriptHover = {
       }
       return undefined;
     } catch (err) {
+      // NOTE - This could happen because of an error in other TS plugins
+      //        I.e., we could end up propagating errors from other plugins
+      //        (This happened to me with the tailwind intellisense plugin)
```

Comment:
I was thinking: Is there a way where we can better expose the exact error the user? 

It seems like we can't show multiline notifications (see https://github.com/microsoft/vscode/issues/101589) but perhaps we could do something where we add a "details" button so the user can find out more information around the exact error (which might help us resolve issues faster)